### PR TITLE
Use black formatting

### PR DESCRIPTION
**Pull request by niklassiemer**

### CODE REVIEW

File: pyiron_atomistics/__init__.py [hunk 1]
```diff
@@ -3,25 +3,41 @@
...
 create_structure = _StructureFactory.crystal
 
 # Make classes available for new pyiron version
 JOB_CLASS_DICT["ART"] = "pyiron_atomistics.interactive.activation_relaxation_technique"
 JOB_CLASS_DICT["AtomisticExampleJob"] = "pyiron_atomistics.testing.randomatomistic"
-JOB_CLASS_DICT["ConvEncutParallel"] = "pyiron_atomistics.dft.master.convergence_encut_parallel"
-JOB_CLASS_DICT["ConvEncutSerial"] = "pyiron_atomistics.dft.master.convergence_encut_serial"
-JOB_CLASS_DICT["ConvergenceVolume"] = "pyiron_atomistics.atomistics.master.convergence_volume"
-JOB_CLASS_DICT["ConvKpointParallel"] = "pyiron_atomistics.dft.master.convergence_kpoint_parallel"
+JOB_CLASS_DICT[
```
Comment (jan-janssen) [inline pyiron_atomistics/__init__.py:29]: Maybe it is more elegant to define a dictionary and then iterate over the dictionary to copy the entries to the `JOB_CLASS_DICT`.

File: pyiron_atomistics/atomistics/structure/_visualize.py [hunk 2]
```diff
@@ -672,16 +687,29 @@ def _get_orientation(view_plane):
...
     """
     if len(np.array(view_plane).flatten()) % 3 != 0:
-        raise ValueError("The shape of view plane should be (N, 3), where N = 1, 2 or 3. Refer docs for more info.")
+        raise ValueError(
+            "The shape of view plane should be (N, 3), where N = 1, 2 or 3. Refer docs for more info."
+        )
     view_plane = np.array(view_plane).reshape(-1, 3)
     rotation_matrix = np.roll(np.eye(3), -1, axis=0)
-    rotation_matrix[:len(view_plane)] = view_plane
+    rotation_matrix[: len(view_plane)] = view_plane
```
Comment (niklassiemer) [inline pyiron_atomistics/atomistics/structure/_visualize.py:695]: Should not harm but looks a bit weird?

**Review by niklassiemer — COMMENTED** [review]
I skimmed over all changes and I did not see any issue with applying black to the codebase.

**Comment by niklassiemer** [comment]
closed in favor of #507

**Pull request closed without merging**